projects/canopy/src/lib/forms/checkbox-group/checkbox-group.stories.ts

```diff
@@ -46,14 +52,58 @@ class ReactiveFormComponent {
     this.form.valueChanges.subscribe(val => this.checkboxChange.emit(val));
   }
 }
+@Component({
+  selector: 'lg-dynamic-form',
+  template: `
+    <form [formGroup]="form">
+      <div
+        *ngFor="let control of changeTypes.controls; index as i"
+        [formGroup]="control"
+        class="change-types__item"
+      >
+        <lg-toggle
+          [formControl]="control.get('selected')"
+          [value]="true"
+          variant="checkbox"
+          >{{ async.content.changeTypes.options[control.get('name')?.value] }}</lg-toggle
```

Comment:
This could just `control.get('name')?.value` for the purposes of this example since the async.content is specific to the project I'm working on